### DISCUSSION TITLE
主旨について

### DISCUSSION THREAD

**Discussion by Honekatsu** [PeaCH-ktq/PeaCH-ktq#8] [General]
サークルの主旨について議論する場所です．

**Comment by Honekatsu** [comment]
「そもそも，サークルの主旨ってどういうものだろう？」となっています

**Comment by ushed** [comment]
大学の講義が想定とずれていた人が、やりたいことができるコミュニティの構築

**Comment by suzuka-y** [comment]
本サークルでは，学術よりも創作に視点を置いたソフトウェア開発を主に行う．本サークルは，大学で履修する内容と入学前の大学に対するイメージに差異が生じ，学術的な学習でなく，自身の創作意欲に従った学習・創作活動を希望する学生に対する支援を行う．本サークルは，ソフトウェア開発に限らず，様々な分野に対する創作意欲を持つ学生が，個人あるいは同じ分野への関心を持つ同志と共同で創作活動を行う．

**Reply by suzuka-y** [reply]
（修正1）

本サークルでは，学術よりも創作に視点を置いたソフトウェア開発を主に行う．本サークルは，大学で履修する内容と入学前の大学に対するイメージに差異が生じ，学術的な学習でなく，自身の創作意欲に従った学習・創作活動を希望する学生に対する支援を行う．加え，本サークルの運営者は，昨今の企業研修において，充実した研修を用意できる企業は減少傾向にある一方で，情報システム工学科の学生の大多数に関しては，卒業後にプログラム開発能力が実務レベルに達していない，という問題を認識している．

本サークルは，ソフトウェア開発に限らず，様々な分野に対する創作意欲を持つ学生が，個人あるいは同じ分野への関心を持つ同志と共同で創作活動を行う．（考えてみただけ，採用はしないかな）

**Reply by suzuka-y** [reply]
（修正2）

本サークルでは，学術よりも創作に視点を置いたソフトウェア開発を主に行う．本サークルは，自身の創作意欲に従った学習・創作活動を希望する学生に対する支援を行う．本サークルは，ソフトウェア開発に限らず，様々な分野に対する創作意欲を持つ学生が，個人あるいは同じ分野への関心を持つ同志と共同で創作活動を行う．
